on-prem-docs/bootstrap-core.md

```diff
@@ -6,6 +6,27 @@ Generate a Personal Access Token for bootstrapping the `core` packages, as well
 
 Select your Gravatar icon on the top right corner of the Chef Habitat Builder on-prem web page, and then select **Profile**. This will take you to a page where you can generate your access token. Make sure to save it securely.
 
+## Add a License Key
+
```

Comment:
I think for continuity with the section above, you could make this section shorter. I would shorten it to:

```
You can enter your license key just below the Personal Access Token text field in order for your account to access official chef maintained packages.
```